lib/routes/sign.js

```diff
@@ -94,33 +94,37 @@ module.exports = function (log, isA, error, signer, db, domain) {
         var deviceId = sessionToken.deviceId ?
           sessionToken.deviceId.toString('hex') : null
 
-        return signer.sign(
-          {
-            email: uid + '@' + domain,
-            publicKey: publicKey,
-            domain: domain,
-            duration: duration,
-            generation: sessionToken.verifierSetAt,
-            lastAuthAt: sessionToken.lastAuthAt(),
-            verifiedEmail: sessionToken.email,
-            deviceId: deviceId
-          }
-        )
-        .then(
-          function(certResult) {
-            log.activityEvent(
-              'account.signed',
-              request,
+        return db.sessionTokenWithVerificationStatus(sessionToken.id)
```

Comment:
> Can we instead change the `sessionTokenWithDevice` auth strategy to look up the verification status as well?

It *should* already work like that with the [most recent DB changes](https://github.com/mozilla/fxa-auth-db-mysql/pull/132/files#diff-4d8dbe6172bcf4632f868ee275eafffcR123).